.github/workflows/build_and_test_cmake.yaml

```diff
@@ -53,12 +53,18 @@ jobs:
       with:
         submodules: true
         ref: ${{ github.event.pull_request.head.sha }}
+        fetch-depth: ${{ !contains(github.event.pull_request.labels.*.name, 'rebase') && 1 || 0 }}
+    - name: Rebase PR
```

Comment:
looks like this commit not just about fixing ternary operator but also about doing rebase ... better to split into separate commits

Reply:
This is just fixing places where rebase should happen, but was missed in previous iteration. It will work weird if you split it in separate changes.

Reply:
Do you mean move this thing to separate commit inside this PR? It will be squashed afterwards though

Reply:
maybe just  change commit name to `adding missing rebase and fixing ternary expressions` or something when you merge so it will be less confusing.
In general you don't have to squash you can "rebase and merge" and preserve two commits if it makes sense to preserve them. Anyway I don't insist on changing this PR. But would be great to have separate commits in the future